tensorflow_2/README.md

```diff
@@ -0,0 +1,45 @@
+# Samples for Notebook to Kubeflow Deployment using TensorFlow 2.0 Keras
+This directory aims at building Kubeflow use case samples with Tensorflow 2.0 Keras training code demonstrating 'Customer User Journey' (CUJ) in the process.
+
+The samples hosted are as follows - 
```

Comment:
Use complete sentences and paragraphs

Reply:
I have updated README. Now it uses complete sentences.